kolibri/core/assets/src/vue/content-icon/index.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+
+  <div>
+    <svg
+      viewbox="0 0 32 32"
+      class="content-icon"
+      :title="altText">
+      <circle
+        cy="16"
+        cx="16"
+        r="16"
+        class="progress-circle"
+        :style="{ 'stroke-dasharray': progressPercent + ' 100' }">
+      </circle>
+      <circle
+        cy="16"
+        cx="16"
+        r="13.5"
+        class="inner-circle">
+      </circle>
+      <svg v-if="thisIs('audio')" src="./content-icons/audio.svg" class="content-type-icon"></svg>
+      <svg v-if="thisIs('document')" src="./content-icons/document.svg" class="content-type-icon"></svg>
+      <svg v-if="thisIs('exercise')" src="./content-icons/exercise.svg" class="content-type-icon"></svg>
+      <svg v-if="thisIs('video')" src="./content-icons/video.svg" class="content-type-icon"></svg>
+    </svg>
+  </div>
+
+</template>
+
+
+<script>
+
+  const KINDS = ['audio', 'document', 'video']; // not 'exercise' for now
```

Comment:
Let's add this to the core constants file like:

```
...

const ContentKinds = {
  AUDIO: 'audio',
  DOCUMENT: 'document',
  VIDEO: 'video',
};

module.exports = {
  UserKinds,
  ContentKinds,
};

```

Then they can be re-used in e.g. the content-renderer.